src/main/java/core/basesyntax/IsoscelesTrapezoid.java

```diff
@@ -0,0 +1,25 @@
+package core.basesyntax;
+
+public class IsoscelesTrapezoid extends AbstractFigure {
+    private String color;
```

Comment:
The `color` field is redundant here because it is already defined in the `AbstractFigure` class. You should remove this field from the `IsoscelesTrapezoid` class to avoid confusion and potential errors.